src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmEvaluationResultFlags.cs

```diff
@@ -31,5 +31,6 @@ public enum DkmEvaluationResultFlags
         CanFavorite = 0x1000000,
         IsFavorite = 0x2000000,
         HasFavorites = 0x4000000,
+        ExpansionHasSideEffects = 0x10000000,
```

Comment:
>0x10000000 [](start = 34, length = 10)

Is the gap intentional or should this be 0x8000000?

Reply:
There are quite a few flags missing from this list, 0x10000000 is the actual value from Concord.

Reply:
I'll update with the full list.